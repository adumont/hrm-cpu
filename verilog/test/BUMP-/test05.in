38 20
01 06
38 c4
22 13
28 4c
06 39
02 2d
03 80
25 cf
43 dd
59 47
24 ef
18 22
18 b2
15 71
59 f2
32 d0
