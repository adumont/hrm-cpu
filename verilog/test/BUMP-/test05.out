1f
05
c3
12
4b
38
2c
7f
ce
dc
46
ee
21
b1
70
f1
cf
